cv/cv/src/cv_utils/camera_projection.py

```diff
@@ -23,51 +26,60 @@ def __init__(self):
             self.depth_values += [self.depth_map[key]]
         self.depth_values = np.array(self.depth_values)
 
+        camera_info = rospy.wait_for_message('/zed/zed_node/rgb/camera_info', CameraInfo)
 
-        '''
-        Raw parameters (found in /usr/local/zed/settings)
-        fx = 1400.13
-        fy = 1400.13
-        cx = 1188.06
-        cy = 645.894
+        # Update camera intrinsics to account for the resized new images
+        scale_x = 330.0/camera_info.width
+        scale_y = 180.0/camera_info.height
```

Comment:
Could you update the height and the width of the camera info too (especially if we are using downstream). Would be good if we add things in the future and for whatever reason rely on the camera info's width/height